extensions/UserProfile/Extension.pm

```diff
@@ -34,6 +35,65 @@ BEGIN {
     *Bugzilla::User::last_statistics_ts       = \&_user_last_statistics_ts;
     *Bugzilla::User::clear_last_statistics_ts = \&_user_clear_last_statistics_ts;
     *Bugzilla::User::address                  = \&_user_address;
+    *Bugzilla::User::getlinks                  = \&_getlinks;
+    *Bugzilla::User::getlinksinfo                  = \&_getlinksinfo;
+}
+
+
+sub _getlinksinfo {
+    my $self = shift;
+    my $value = Bugzilla->dbh->selectrow_array(
+            "SELECT linksinfo FROM profiles WHERE userid = ?",
+            undef,
+            $self->id
+        );
+    return $value;
+}
+
+sub _getlinks {
+    my $self = shift;
+    my $value = $self->getlinksinfo();
+
+    my $state = 1;
+
+    my $hash;
+    my $arr = [ ];
+
+    for my $elem (split (/\n/, $value)) {
```

Comment:
This is clever!

However I think we want only a single URL on the profile, and that the URL should be to "Mozillians"